migration/20160614-add-coverage-status.sql

```diff
@@ -10,8 +10,8 @@ alter table workcoveragerecords add column status coverage_status;
 CREATE INDEX ix_coveragerecords_status ON coveragerecords USING btree (status);
```

Comment:
This is an unrelated change I decided I might as well commit. I had to rerun this migration script and this change makes it run a lot faster the second time.